### ISSUE TITLE
JetReviews: filling out name & email is missing as guest in 2.1.0 update

### ISSUE THREAD

**Issue by Medienor** [CrocoBlock/suggestions#2716]
In JetReviews 2.1.0 update we was told it should be now a option for adding your name and email when leaving a review as a guest.

It's still not possible

https://prnt.sc/107h099

When can somebody have a look at this?
I know their are other users experencing same problems.

**Comment by Bookit24** [comment]
Same issue here. I was waiting for this functionality and updated the plugin as soon as it was released, but it's not working. 
The name/email is saved to the db, but it still just show as "guest" in "all reviews" and in the posts where the review has been left. 

I've had a support ticket open for 15 days regarding this, but it has not been resolved yet. It is quite frustrating that it's not being solved quicker when it was reported just after the release.

In my case, I hope it's being solved quickly now and that I will be able to use it - but I've started testing other plugins to see if they should be used instead. It makes me worried to use a plugin that will be a critical part of my website when it takes this long to solve reported issues.

This has also been reported in https://github.com/Crocoblock/suggestions/issues/2668
